tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -42,6 +42,11 @@ using ::nvinfer1::IRuntime;
 using ::tensorflow::strings::StrAppend;
 using ::tensorflow::strings::StrCat;
 
+std::size_t VectorTensorShapeHasher::operator()(
+    const std::vector<tensorflow::TensorShape>& k) const {
+  return std::hash<std::string>()(TensorShapeUtils::ShapeListString(k));
```

Comment:
@aaroey @smit-hinsu Is there a better hash function for vector\<TensorShape\> we should use instead?